diff --git a/node_modules/react-native/ReactAndroid/build.gradle b/node_modules/react-native/ReactAndroid/build.gradle
index f44b6e4..2520ab0 100644
--- a/node_modules/react-native/ReactAndroid/build.gradle
+++ b/node_modules/react-native/ReactAndroid/build.gradle
@@ -431,8 +431,8 @@ repositories {
 }
 
 android {
-    buildToolsVersion = "33.0.0"
-    compileSdkVersion 33
+    buildToolsVersion = "34.0.0"
+    compileSdkVersion 34
     namespace "com.facebook.react"
 
     // Used to override the NDK path/version on internal CI or by allowing
